modules/nw-dns-cache-tuning.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+// * networking/dns-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-dns-cache-tuning_{context}"]
+= Tuning the CoreDNS cache
+
+You can configure the maximum duration of both successful (positive) or unsuccessful (negative) caching done by CoreDNS. Tuning the duration of caching of DNS query responses can reduce the load for any upstream DNS resolvers.
```

Comment:
We generally avoid parenthesis per IBM style guide.

Reply:
@jboxman-rh what do you think about the following as a replacement for the parentheses?

> You can configure the maximum duration of both successful or unsuccessful caching, otherwise known as positive or negative caching respectively, done by CoreDNS.

I'm not too sure it flows any better than before, but I'm struggling a bit to come up with a way to mention the alternative phrasing without the parentheses